common/setups/rasr/config/am_config.py

```diff
@@ -0,0 +1,142 @@
+__all__ = ["Tdp", "SpeechTdp", "SilenceTdp", "NonSpeechTdp", "AmRasrConfig"]
+
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import List, Optional, Tuple, Union
+
+
+from sisyphus import tk
+
+import i6_core.rasr as rasr
+
+
+class StateTying(Enum):
+    NONE = "none"
+    DECISION_TREE = "decision-tree"
+    CART = "cart"
+    MONOPHONE = "monophone"
+    MONOPHONE_EOW = "monophone-eow"
+    MONOPHONE_DENSE = "monophone-dense"
+    DIPHONE_DENSE = "diphone-dense"
+    NO_TYING_DENSE = "no-tying-dense"
+    NO_TYING = "no-tying"
+    LUT = "lut"
+    LOOKUP = "lookup"
+
+    def __str__(self):
+        return self.value
+
+
+@dataclass()
+class Tdp:
```

Comment:
It is true that for Viterbi alignment and decoding we generally use the non-normalized `default` tdp values, however, for full-sum training, we generally introduce normalized loop/forward values that might be even estimated from an alignment or defined based on aomse heuristics, e.g. average phoneme length. In addition to this class, we could also introduce an `enum` class for `TdpType`, out of `default`, `heuristic`,  and `alignment-based`. For the two latter types one could also have jobs that estimate values from an alignment or from transcription. Daniel Mann has already such jobs.

Reply:
Please address also the other comment, before merging the current commit. In my honest opinion, that type of search over the parameters is not correct.

Reply:
I created two new issues.
Currently you have a lot of flexibility, I rather like that...